callstats-java-sdk/src/main/java/io/callstats/sdk/CallStatsAsyncHttpClient.java

```diff
@@ -0,0 +1,81 @@
+package io.callstats.sdk;
+
+
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import com.ning.http.client.AsyncCompletionHandler;
+import com.ning.http.client.AsyncHttpClient;
+import com.ning.http.client.AsyncHttpClientConfig;
+import com.ning.http.client.AsyncHttpClientConfig.Builder;
+import com.ning.http.client.Request;
+import com.ning.http.client.RequestBuilder;
+import com.ning.http.client.Response;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CallStatsAsyncHttpClient {
+	
+	private String BASE_URL;
+	private int connectionTimeOut;
+	private static final Logger logger = LogManager.getLogger("CallStatsAsyncHttpClient");
+	private AsyncHttpClient httpClient;
+
+	public CallStatsAsyncHttpClient() {
+		Properties prop = new Properties();
+		InputStream input = null;
+	     
+    
+		input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+		if(input !=null){
+			try {
+				prop.load(input);
+				BASE_URL = prop.getProperty("CallStats.BaseURL");
+				connectionTimeOut = Integer.parseInt(prop.getProperty("CallStats.ConnectionTimeOut"));
+			} catch (IOException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
```

Comment:
This is a general comments to the occurrence of "e.printStackTrace();", you should really continue to throw this exception out instead of "eating" it by only logging to the standard error.